portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,71 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+/*NAVBAR*/
+
+.navbar {
+  overflow: hidden;
+  background-color: #333;
+  top: 10;
+  /*potion: fixed;*/
+  z-index: 2;
+}
```

Comment:
define the css properties in alphabetical order, here and throughout the file. The advantage of this is that,  it is well organized and easy to spot specific css properties within a rule.

Reply:
done